cel/library_test.go

```diff
@@ -26,7 +27,9 @@ import (
 func TestCELLib(t *testing.T) {
 	t.Parallel()
 
-	env, err := cel.NewEnv(cel.Lib(library{}))
+	env, err := cel.NewEnv(cel.Lib(library{
```

Comment:
Should we switch this to use `NewLibrary` as well?

Reply:
I thought that too. Wasn't sure if it was instantiated like this for a reason.

Reply:
Updated. 👍